Packs/Logzio/Integrations/Logzio/README.md

```diff
@@ -0,0 +1,742 @@
+## Overview
+---
+
+Fetch incidents from Logz.io Cloud SIEM to activate Demisto Playbooks
+
+## Use Cases
+---
+
+Integrate with Logz.io Cloud SIEM to automatically remediate security incidents identified by Logz.io and increase observability into incident details. 
```

Comment:
what is the remediation refer to ?
please use `Cortex XSOAR` instead of `Demisto`